appservice/src/createAppService/AppServicePlanListStep.ts

```diff
@@ -36,7 +43,12 @@ export class AppServicePlanListStep extends AzureWizardPromptStep<IAppServiceWiz
 
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
         // Cache hosting plan separately per subscription
-        const options: IAzureQuickPickOptions = { placeHolder: localize('selectPlan', 'Select a {0} App Service plan.', getWebsiteOSDisplayName(nonNullProp(wizardContext, 'newSiteOS'))), id: `AppServicePlanListStep/${wizardContext.subscriptionId}` };
+        const options: IAzureQuickPickOptions = {
+            placeHolder: wizardContext.newSiteKind.includes(AppKind.workflowapp) && wizardContext.planSkuFamilyFilter?.test('IV2')
```

Comment:
I don't remember this filter being here before.  I hadn't seen any `IV2` plans-- just wanted to make sure that this was intentional.

Reply:
Yes this was intentional, today we only support the new sku for workflow and only one specific type of sku, this is for that and only specific to workflow app. I can explain more in teams